components/organisms/ContributorProfileTab/contributor-profile-tab.tsx

```diff
@@ -137,7 +136,8 @@ const ContributorProfileTab = ({
             className={clsx(
               "data-[state=active]:border-sauced-orange data-[state=active]:border-b-2 text-2xl",
               tab === "Recommendations" &&
-                " data-[state=active]:font-bold data-[state=active]:text-transparent data-[state=active]:bg-clip-text data-[state=active]:bg-gradient-to-r data-[state=active]:from-[#EA4600] data-[state=active]:to-[#EB9B00]"
+                " data-[state=active]:font-bold data-[state=active]:text-transparent data-[state=active]:bg-clip-text data-[state=active]:bg-gradient-to-r data-[state=active]:from-[#EA4600] data-[state=active]:to-[#EB9B00]",
+              user && user.user_metadata.user_name !== login && tab === "Recommendations" && "hidden"
```

Comment:
The recommendation tab won't show up if I'm logged in but will if I'm not. You should also apply the `hidden` class if there is no user 🍕 

![recom-2](https://github.com/open-sauced/insights/assets/79809121/0607340d-369f-4515-b1a2-49aa24035c91)